pyproject.toml

```diff
@@ -1,12 +1,53 @@
-[tool.coverage.run]
-omit = [
-    "viadot/flows/azure_sql_transform_dbt.py",
-    "viadot/flows/supermetrics_to_adls.py",
-    "viadot/flows/supermetrics_to_azure_sql_v2.py"
-    ]
+[project]
+name = "viadot2"
+version = "2.0a21"
```

Comment:
Is the package version taken from here now? If so, shouldn't the version be removed from https://github.com/dyvenia/viadot/blob/d1c0441289f2a5db18878f9dd771d17b2951971b/src/viadot/__init__.py#L3?

Also seems like this can be removed

https://github.com/dyvenia/viadot/blob/2.0/.github/workflows/release.yml#L90-L102

The release process is not documented but the only required thing I think was to update the version in `viadot__init__.py` (and then the PyPI release action could be manually triggered); does this PR alter this process? Can you document the release process in `CONTRIBUTING.md` please? I think it's literally just one paragraph, with just these 2 steps, but maybe double-check that it's correct.

Reply:
[🔥 Deleted __version__ from viadot/__init__.py](https://github.com/dyvenia/viadot/pull/878/commits/e19c57db3b37703a6eabaac31c8c36251e3ae1ce)

Reply:
I prefer the document release process after updating and checking all github actions in the repository. Let's first merge this PR to not have it too big.